reducers/views/channel_selector_modal.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {ChannelTypes, UserTypes} from 'mattermost-redux/action_types';
+
+export default function channelsForChannelSelector(state = {}, action) {
```

Comment:
0/5 since I doubt this'll actually matter, but it might make sense with this to have this reducer be nested one level deeper like `state.views.channelSelectorModal.channels` in case we were ever to add more redux state to this modal. I personally default to starting a reducer file that way, but it's not like it would be difficult to change to that if we find we need it

Reply:
Done!